.conda/meta.yaml

```diff
@@ -1,16 +1,20 @@
 {% set name = "FenicsXConcrete" %}
-{% set version = "0.0.0" %}
+# For setting the version, either tag your git commit,
+# or set an environment variable GIT_DESCRIBE_TAG
 
 package:
   name: {{ name|lower }}
-  version: {{ version }}
+  version: 0.0.1
```

Comment:
Shouldn't we have the version being automatically set by the git_tag?

Reply:
yes, I added a ToDo